code/controllers/CMSPageHistoryController.php

```diff
@@ -358,7 +358,7 @@ public function ShowVersionForm($versionID = null) {
 	/**
 	 * @param int $versionID
 	 * @param int $otherVersionID
-	 * @return Form
+	 * @return mixed Form|boolean|SS_HTTPResponse
```

Comment:
this syntax is slightly wrong. it should either be `mixed` OR `Form|bool`

I don't think this ever returns a `SS_HTTPResponse` object because `Security::permissionFailure` throws an exception rather than returns anything?! IIRC...